components/textbook-beta/TextbookBetaHeader.vue

```diff
@@ -131,6 +125,35 @@ export default class TextbookBetaHeader extends (Vue as VueConstructor<VueCompon
     }
   }
 
+  &__logo {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    width: 100%;
+    padding-top: 100%;
+    transform: translate(-50%, -50%);
+
+    &-container {
+      // the #{_______} is added to force scss to compile $spacing-11 value.
```

Comment:
IMO this comment is not needed. Sass interpolation is well documented on their docs: https://sass-lang.com/documentation/interpolation

Reply:
This was a @y4izus request since `--additional-width: $spacing-11;` this doesn't work as expected.